modules/ding_session_cache/ding_session_cache.module

```diff
@@ -111,9 +111,12 @@ function ding_session_cache_get($module, $cid, $default = FALSE) {
  *   Name of the module that the date is stored under (prefix).
  * @param string $cid
  *   Index data is stored under.
+ * @param bool $wildcard
+ *   If TRUE, the $cid argument must contain a string value and cache IDs
+ *   starting with $cid are deleted.
```

Comment:
This comment is somewhat misleading. According to the definition of `$cid` it must be a string. `NULL` is not allowed although it is supported by `cache_clear_all()`.

Reply:
The comment is copied from Drupal core!

Reply:
Fixed

Reply:
> The comment is copied from Drupal core!

Copy/pasting from Drupal Core does not make it right. Drupal Core does not specify that `$cid` is a string. We do. It is OK for us to set stricter requirements. Then we have to follow up on them ourselves.